corehq/apps/data_analytics/views.py

```diff
@@ -0,0 +1,9 @@
+from corehq.apps.data_analytics.models import MALTRow
+from corehq.apps.data_analytics.admin import MALTRowAdmin
+from dimagi.utils.django.management import export_as_csv_action
+
+
+def malt_csv_response(month, year):
```

Comment:
er - just kidding, saw the real view is below. what's the purpose of this file? seems like this code can just live in `hqadmin.views`